### ISSUE TITLE
KeyError: 'saltutil.is_running' on Fedora 31

### ISSUE THREAD

**Issue by furlongm** [saltstack/salt#55787]
### Description of Issue

Getting `KeyError: 'saltutil.is_running'` on Fedora 31

### Steps to Reproduce Issue
```
$ salt-call --local state.highstate
[WARNING ] /usr/lib/python3.7/site-packages/salt/utils/path.py:265: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  if not isinstance(exes, collections.Iterable):

[WARNING ] /usr/lib/python3.7/site-packages/salt/utils/decorators/signature.py:31: DeprecationWarning: `formatargspec` is deprecated since Python 3.5. Use `signature` and the `Signature` object directly
  *salt.utils.args.get_function_argspec(original_function)

[WARNING ] /usr/lib/python3.7/site-packages/salt/utils/decorators/signature.py:31: DeprecationWarning: `formatargspec` is deprecated since Python 3.5. Use `signature` and the `Signature` object directly
  *salt.utils.args.get_function_argspec(original_function)

[WARNING ] /usr/lib/python3.7/site-packages/salt/utils/decorators/signature.py:31: DeprecationWarning: `formatargspec` is deprecated since Python 3.5. Use `signature` and the `Signature` object directly
  *salt.utils.args.get_function_argspec(original_function)

[ERROR   ] An un-handled exception was caught by salt's global exception handler:
KeyError: 'saltutil.is_running'
Traceback (most recent call last):
  File "/usr/bin/salt-call", line 11, in <module>
    salt_call()
  File "/usr/lib/python3.7/site-packages/salt/scripts.py", line 431, in salt_call
    client.run()
  File "/usr/lib/python3.7/site-packages/salt/cli/call.py", line 57, in run
    caller.run()
  File "/usr/lib/python3.7/site-packages/salt/cli/caller.py", line 138, in run
    ret = self.call()
  File "/usr/lib/python3.7/site-packages/salt/cli/caller.py", line 237, in call
    ret['return'] = self.minion.executors[fname](self.opts, data, func, args, kwargs)
  File "/usr/lib/python3.7/site-packages/salt/executors/direct_call.py", line 12, in execute
    return func(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 1023, in highstate
    conflict = _check_queue(queue, kwargs)
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 419, in _check_queue
    conflict = running(concurrent=kwargs.get('concurrent', False))
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 378, in running
    active = __salt__['saltutil.is_running']('state.*')
  File "/usr/lib/python3.7/site-packages/salt/loader.py", line 1208, in __getitem__
    func = super(LazyLoader, self).__getitem__(item)
  File "/usr/lib/python3.7/site-packages/salt/utils/lazy.py", line 108, in __getitem__
    raise KeyError(key)
KeyError: 'saltutil.is_running'
Traceback (most recent call last):
  File "/usr/bin/salt-call", line 11, in <module>
    salt_call()
  File "/usr/lib/python3.7/site-packages/salt/scripts.py", line 431, in salt_call
    client.run()
  File "/usr/lib/python3.7/site-packages/salt/cli/call.py", line 57, in run
    caller.run()
  File "/usr/lib/python3.7/site-packages/salt/cli/caller.py", line 138, in run
    ret = self.call()
  File "/usr/lib/python3.7/site-packages/salt/cli/caller.py", line 237, in call
    ret['return'] = self.minion.executors[fname](self.opts, data, func, args, kwargs)
  File "/usr/lib/python3.7/site-packages/salt/executors/direct_call.py", line 12, in execute
    return func(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 1023, in highstate
    conflict = _check_queue(queue, kwargs)
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 419, in _check_queue
    conflict = running(concurrent=kwargs.get('concurrent', False))
  File "/usr/lib/python3.7/site-packages/salt/modules/state.py", line 378, in running
    active = __salt__['saltutil.is_running']('state.*')
  File "/usr/lib/python3.7/site-packages/salt/loader.py", line 1208, in __getitem__
    func = super(LazyLoader, self).__getitem__(item)
  File "/usr/lib/python3.7/site-packages/salt/utils/lazy.py", line 108, in __getitem__
    raise KeyError(key)
KeyError: 'saltutil.is_running'
```
### Versions Report
```
Salt Version:
           Salt: 2019.2.1rc0

Dependency Versions:
           cffi: 1.12.3
       cherrypy: Not Installed
       dateutil: 2.8.0
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10.1
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.35.2
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.1
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: Not Installed
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.7.4 (default, Aug 12 2019, 14:45:07)
   python-gnupg: Not Installed
         PyYAML: 5.1.2
          PyZMQ: 18.0.2
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 6.0.2
            ZMQ: 4.3.2

System Versions:
           dist: fedora 31 Thirty One
         locale: UTF-8
        machine: x86_64
        release: 5.3.7-301.fc31.x86_64
         system: Linux
        version: Fedora 31 Thirty One
```

**Comment by frogunder** [comment]
@furlongm Thank you for reporting this issue.

I am however not able to replicate it.

How did you install salt? 

And where did you get the `2019.2.1rc0` salt version from? 
We do not have that version as part of the Saltstack repo in GIthub.

Thanks.

**Comment by furlongm** [comment]
I installed via salt-bootstrap.

**Comment by furlongm** [comment]
It seems this is due to an outdated version in the upstream fedora repos:

```
# dnf provides salt
Last metadata expiration check: 0:14:06 ago on Tue Jan  7 16:04:22 2020.
salt-2019.2.1rc0-3.fc31.noarch : A parallel remote execution system
Repo        : @System
Matched from:
Provide    : salt = 2019.2.1rc0-3.fc31

salt-2019.2.1rc0-3.fc31.noarch : A parallel remote execution system
Repo        : fedora
Matched from:
Provide    : salt = 2019.2.1rc0-3.fc31
```

**Comment by furlongm** [comment]
Upstream bug report: https://bugzilla.redhat.com/show_bug.cgi?id=1785489

**Comment by sagetherage** [comment]
@dmurphy18 are you able to follow up here?

**Comment by sagetherage** [comment]
Assigned to DMurphy and will follow up with him after today - he is working on something else, today, but want to get his eyes on this sooner rather than later.

**Comment by dmurphy18** [comment]
@furlongm Packaged Salt does not currently work on Fedora 31 Py 3 due to Tornado 5 / 6 compatibility issues. However I shall be packaging the latest release Salt 3000 soon which should address this issue.

In the mean time you can bootstrap install latest to install the latest version of Salt, (bootstrapping version 3000 had a bug which was recently fixed and is in the develop branch but not released yet, to my knowledge).   The version 2019.2.1rc0 was an interim release candidate version, and the full 2019.2.1 release was found to contain significant issues, such that it's use was not recommended and a subsequent 2019.2.2 was released with fixes quickly, but no Fedora version was released associated with 2019.2.2.

If this information is sufficient, please consider closing this issue.
